src/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+    <link href="https://fonts.googleapis.com/css2?family=Salsa&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <main class="all">
+      <div class="mainarea">
+        <h1 class="maintitle">Leaderboard</h1>
+      </div>
+      <div class="leaderboard">
+        <h3 class="scores">Recent scores</h3>
+        <button class="refreshBtn">Refresh</button>
+      </div>
+      <div class="leaderboardscores">
+          <div class="score">Name:100</div>
+          <div class="score">Name:69</div>
+          <div class="score">Name:107</div>
+          <div class="score">Name:93</div>
+          <div class="score">Name:40</div>
+          <div class="score">Name:230</div>
+          <div class="score">Name:145</div>
+      </div>
+      <form>
+        <p class="formtitle">Add your score</p>
```

Comment:
- [ ] The same concept of better semantic tags applies here: If this represents a title, wouldn't it be better to use any of the heading `<hx>` tags instead? Please consider changing this one! 🤓 🔧 

```suggestion
        <h2 class="formtitle">Add your score</h2>
```